### ISSUE TITLE
Cannot apply patch for 0001-Yarn-Add-implementation-for-ppc64-ELFv2.patch

### ISSUE THREAD

**Issue by jessesimpson36** [shawnanastasio/chromium_power#1]
Hello,

It looks like the patch for the SwiftShader cannot be applied because the directories within the chromium source directories do not exist. The rest of the patches apply fine to me. 

As you can see in the lines below, I could not find any files that correspond to the files updated in the patch.

I knew to go to chromium/src/third_party/swiftshader because that is the file location that is in patches.json

```
jesse@compiler:~/chromium/build/chromium/src/third_party/swiftshader$ tree | grep YARN
jesse@compiler:~/chromium/build/chromium/src/third_party/swiftshader$ tree | grep Yarn
jesse@compiler:~/chromium/build/chromium/src/third_party/swiftshader$ tree | grep OSFiber
jesse@compiler:~/chromium/build/chromium/src/third_party/swiftshader$ tree | grep BUILD.gn
```

I will attempt to go through the rest of the porting instructions located here:
https://wiki.raptorcs.com/wiki/Porting/Chromium_(New) 

and report back if the rest of the install works.

**Comment by shawnanastasio** [comment]
Hello,

That particular patch was actually upstreamed: https://github.com/google/marl/commit/d40d56e180b3bcaa738996a3ce14c5a1ba04de20. (yarn was renamed to marl and moved to a separate repo).

I'll remove that patch from the repo in an upcoming update, but for now you can safely skip it.

**Comment by jessesimpson36** [comment]
Thanks for letting me know. I've been going through the rest of the compilation process, and I don't think the errors I'm currently getting is related to this patch so you are free to close this issue.

**Comment by jessesimpson36** [comment]
I just realized I could close this issue, but was unsure if I should until this patch gets removed from the tree. If my compilation works, I'll open a patch.

**Comment by shawnanastasio** [comment]
Fixed in e6b95c7da286eb76b21ed59556822a633e4837e9
